common/src/main/resources/memoryleakfix.mixins.json

```diff
@@ -1,27 +1,28 @@
-{
-  "required": true,
-  "minVersion": "0.8",
-  "package": "ca.fxco.memoryleakfix.mixin",
-  "compatibilityLevel": "JAVA_17",
-  "mixins": [
-    "biomeTemperatureLeak.Biome_threadLocalMixin",
-    "entityMemoriesLeak.Brain_clearMemoriesMixin",
-    "entityMemoriesLeak.LivingEntity_clearMemoriesMixin"
-  ],
-  "server": [
-    "spongePoweredLeak.MinecraftServer_loadWorldMixin"
-  ],
-  "client": [
-    "customPayloadLeak.CustomPayloadS2CPacket_bufferMixin",
-    "customPayloadLeak.MinecraftClient_freeBufferMixin",
-    "customPayloadLeak.PacketByteBuf_extendMixin",
-    "hugeScreenshotLeak.MinecraftClient_screenshotMixin",
-    "readResourcesLeak.TextureUtil_freeBufferMixin",
-    "spongePoweredLeak.Main_clientLoadedMixin",
-    "targetEntityLeak.MinecraftClient_targetClearMixin"
-  ],
-  "injectors": {
-    "defaultRequire": 1,
-    "maxShiftBy": 2
-  }
-}
+{
+  "required": true,
+  "minVersion": "0.8",
+  "package": "ca.fxco.memoryleakfix.mixin",
+  "compatibilityLevel": "JAVA_17",
+  "client": [
+    "customPayloadLeak.MinecraftClient_freeBufferMixin",
+    "hugeScreenshotLeak.MinecraftClient_screenshotMixin",
+    "readResourcesLeak.TextureUtil_freeBufferMixin",
+    "spongePoweredLeak.MinecraftClient_clientLoadedMixin",
+    "targetEntityLeak.MinecraftClient_targetClearMixin"
+  ],
+  "server": [
+    "spongePoweredLeak.MinecraftServer_loadWorldMixin"
+  ],
+  "mixins": [
+    "biomeTemperatureLeak.Biome_threadLocalMixin",
+    "customPayloadLeak.CustomPayloadS2CPacket_bufferMixin",
+    "customPayloadLeak.PacketByteBufAccessor",

```

Comment:
These need to be moved to `client`
They may also need `@Environment(EnvType.CLIENT)` if they don't already have it